frameworks.mjs

```diff
@@ -235,4 +235,29 @@ export default [
       return files;
     },
   },
+  {
+    id: "aurelia1",
+    title: "Aurelia 1",
+    img: "framework/aurelia.svg",
+    eslint: {
+      env: {
+        browser: true,
+        es2021: true,
+        node: true,
+      },
+      parser: "@typescript-eslint/parser",
+      parserOptions: {
+        ecmaFeatures: {
+          jsx: true,
+        },
+      },
+      files: ["**/aurelia1/**"],
+      extends: ["eslint:recommended"],
+    },
+    playgroundURL: "https://codesandbox.io/s/ppmy26opw7",
+    documentationURL: "http://aurelia.io/docs/",
+    filesSorter(files) {
+      return files;
```

Comment:
Specifying a files sorter would be great for the user to follow the path on how multiple components are working together. For example, "app.*" files must be first to display.
The other frameworks got great examples.

Reply:
I don't think other files are needed, besides the app.html and app.ts. So updated accordingly. Nice addition! I did think about the order, but was not aware of this option.